pypeit/core/pydl.py

```diff
@@ -430,6 +431,7 @@ def func_fit(x, y, ncoeff, invvar=None, function_name='legendre', ia=None,
     return res, yfit
 
 
+# TODO -- This class needs to become a DataContainer
```

Comment:
Traceset is nothing more than a list/array of fits, so we could add a ToDO that at some point this should be merged somehow with your pypeitFit object, or you need a new object that represents a list/array of these. Anyway, I don't think we write Traceset to disk, so unclear whether it needs to be a DataContainer.